rpcs3/rpcs3qt/debugger_list.h

```diff
@@ -16,24 +16,28 @@ class debugger_list : public QListWidget
 {
 	Q_OBJECT
 
+	u32 m_start_addr = 0;
```

Comment:
why isn't this in the member section at the bottom?

Reply:
It's private.

Reply:
there already is a private section at the bottom. It makes no sense to add this here.